geometries/ramangle/ramangle_algorithm.py

```diff
@@ -1,77 +1,42 @@
-import math as m
 import spiceypy as sp
 
-def ramangle(utc, target, galaxy_targ):
-    """Calculates ram angle, which is measured between velocity vector of Aspera & vector from Aspera
-    to galaxy.
+def ramangle(utc, instr):
+    """Calculates ram angle, which is measured between Earth-relative
+    velocity vector of Aspera & boresight vector of instrument
 
     Args:
-        mkfile (str): metakernel containg data on Aspera, nearby bodies, and M82
         utc (str): date and time at which altitude will be found
-        galaxy_targ (str): body ID for galaxy contained in mkfile
+        instr (str): instrument from which to extract frame & boresight
 
     Returns:
         float: ram angle
     """
 
-    # # # # # PART 1: ANGLE BETWEEN VELOCITY AND POSITION VECTORS # # # # #
+    # # # # # PART 1: ANGLE BETWEEN EARTH-RELATIVE VELOCITY AND BORESIGHT VECTORS # # # # #
 
     et = sp.str2et(utc)
-    ref = 'J2000'
     abcorr = 'NONE'
 
+    #btc get instrument frame and borsight, also instrument frame center
+    ref,vbore = sp.getfov(sp.bods2c(instr),99,99,99)[1:3]
```

Comment:
it is simpler to get all vectors in the instrument's reference frame
- i.e. pass instrument frame to sp.spkezr(...) call
- then all vectors will be in the same frame as the boresight